tests/e2e/macvtap_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests_test
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strconv"
+)
+
+var _ = Describe("macvtap-cni", func() {
+
+	Describe("macvtap resource creation", func() {
```

Comment:
Just seeing how this specs could read better, I don't think this level adds much.

Reply:
Now that I've added the other context (check that make cluster-sync starts the daemonsets) maybe it does. 

I'm not exactly sold on testing that... But this was a good 1st exercise.

Another thing I'm not exactly sold on here is that we're relying on the configuration performed by the config map in the 'examples' / generated ds manifest . That's far from bullet proof.

I plan on updating that *once* we work on having the ds react to configuration updates. We should sync on that.